test.js

```diff
@@ -209,3 +210,18 @@ test('fallback to cwd if `module.filename` is `null`', t => {
 	module.filename = preservedFilename;
 	del.sync(conf.path, {force: true});
 });
+
+// Test encryption.
+test('.get() (encrypted)', t => {
```

Comment:
`'.get() (encrypted)'` => `'.get() - encryption'`

Reply:
Actually, I don't think you need two tests for this almost doing the same thing, just test `get` and `set` in the same test.